docs/overview.yaml

```
@@ -320,13 +320,44 @@ Analysis:
     monotone convergence theorem: 'measure_theory.lintegral_infi_ae'
     Fatou's lemma: 'measure_theory.lintegral_liminf_le'
     vector-valued integrable function (Bochner integral): 'measure_theory.integrable'
+    uniform integrability: 'measure_theory.uniform_integrable'
     $L^p$ space: 'measure_theory.Lp'
     Bochner integral: 'measure_theory.integral'
     dominated convergence theorem: 'measure_theory.tendsto_integral_of_dominated_convergence'
     fundamental theorem of calculus, part 1: 'interval_integral.integral_has_fderiv_at_of_tendsto_ae'
     fundamental theorem of calculus, part 2: 'interval_integral.integral_eq_sub_of_has_deriv_right_of_le'
     Fubini's theorem: 'measure_theory.integral_prod'
     product of finitely many measures: 'measure_theory.measure.pi'
+    convolution: 'convolution'
+    approximation by convolution: 'cont_diff_bump_of_inner.convolution_tendsto_right'
+    regularization by convolution: 'has_compact_support.cont_diff_convolution_left'
+
+Probability Theory:
+  Definitions of a probability space:
+    probability measure: 'measure_theory.is_probability_measure'
+    independent events: 'probability_theory.Indep_set'
+    independent sigma-algebra: 'probability_theory.Indep'
+    conditional probability: 'probability_theory.cond'
+  Random variables and their laws:
+    discrete law: 'pmf'
+    probability density function: 'measure_theory.has_pdf'
+    variance of a real-valued random variable: 'probability_theory.variance'
+    independence of random variables: 'probability_theory.Indep_fun'
+    mean of product of independent random variables: 'probability_theory.indep_fun.integral_mul_of_integrable'
+    moment of a random variable: 'probability_theory.moment'
+    Bernoulli law: 'pmf.bernoulli'
+  Convergence of a sequence of random variables:
+    convergence in probability: 'measure_theory.tendsto_in_measure'
+    $\mathrm{L}^p$ convergence: 'measure_theory.Lp'
+    almost surely convergence: 'measure_theory.measure.ae'
+    Markov inequality: 'measure_theory.mul_meas_ge_le_lintegral'
```

We have convergence in distribution (weak convergence) although much of the API (in particular portmanteau theorem) is still on the way.

```suggestion
    convergence in distribution: 'measure_theory.probability_measure.tendsto_iff_forall_integral_tendsto'
    Markov inequality: 'measure_theory.mul_meas_ge_le_lintegral'
```